packages/components/src/components/text-input/_text-input.scss

```diff
@@ -142,7 +142,7 @@
   //-----------------------------
   .#{$prefix}--text-input:disabled {
     cursor: not-allowed;
-    outline: none;
+    outline: 2px solid transparent;
```

Comment:
`@include focus-outline('reset');`?

Reply:
Yes! 🤦 

Updated, thanks for catching that.